Tasks/NuGetCommand/task.loc.json

```diff
@@ -322,6 +322,28 @@
       "visibleRule": "versioningScheme = byPrereleaseNumber"
     },
     {
+      "name": "packTimezone",
+      "type": "pickList",
+      "label": "ms-resource:loc.input.label.packTimezone",
+      "defaultValue": "utc",
+      "helpMarkDown": "ms-resource:loc.input.help.packTimezone",
+      "required": "false",
+      "groupName": "packOptions",
+      "options": {
+        "utc": "UTC",
```

Comment:
The internal keys are not localized since they are not user visible, and it seems the values are not localized. This is handled by the build team.